Git Formats/Git Config.sublime-syntax

```diff
@@ -47,7 +47,11 @@ contexts:
     - meta_scope: meta.brackets.git.config
     - match: \]
       scope: punctuation.definition.brackets.end.git.config
-      pop: 1
+      set:
+        - match: \s*$
+          scope: meta.whitespace.newline.git.config
+        - match: ''
+          pop: 1
```

Comment:
What should be added is a dedicated meta scope for section headers. INI syntax uses `meta.section`. So we could rename `meta.brackets` to somewhat more specific:

```yml
  section-header:
    - match: \[
      scope: punctuation.definition.section.begin.git.config
      set: [expect-line-end, section-header-end, section-name]

  section-header-end:
    - meta_scope: meta.section.git.config
    - match: \]
      scope: punctuation.definition.section.end.git.config
      pop: 1
    - include: illegal-line-end-pop
    - match: \S
      scope: invalid.illegal.header-end.git.config
```